File: app/src/main/java/com/BlizzardArmory/overwatch/Rating.java
package com.BlizzardArmory.overwatch;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Rating {

    @SerializedName("level")
    @Expose
    private int level;
    @SerializedName("role")
    @Expose
    private String role;
    @SerializedName("roleIcon")
    @Expose
    private String roleIcon;
    @SerializedName("rankIcon")
    @Expose
    private String rankIcon;

    public int getLevel() {
        return level;
    }

    public void setLevel(int level) {
        this.level = level;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }

    public String getRoleIcon() {
        return roleIcon;
    }

    public void setRoleIcon(String roleIcon) {
        this.roleIcon = roleIcon;
    }

    public String getRankIcon() {
        return rankIcon;
    }

    public void setRankIcon(String rankIcon) {
        this.rankIcon = rankIcon;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/overwatch/Rating.kt
package com.BlizzardArmory.model.overwatch

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * The type Rating.
 */
data class Rating(

        @SerializedName("level")
        @Expose
        var level: Int,

        @SerializedName("role")
        @Expose
        var role: String,

        @SerializedName("roleIcon")
        @Expose
        var roleIcon: String,

        @SerializedName("rankIcon")
        @Expose
        var rankIcon: String

)